js_modules/dagit/packages/core/src/runs/LogsProvider.tsx

```diff
@@ -159,12 +162,29 @@ const useLogsProviderWithSubscription = (runId: string) => {
   // Batch the nodes together so they don't overwhelm the animation of the Gantt,
   // which depends on a bit of a timing delay to maintain smoothness.
   const throttledSetNodes = React.useMemo(() => {
-    return throttle((hasMore: boolean, cursor: string) => {
-      const queued = [...queue.current];
+    return throttle(() => {
+      if (!queue.current.length) {
+        return;
+      }
+      const queuedLogs = [...queue.current];
       queue.current = [];
-      dispatch({type: 'append', queued, hasMore, cursor});
+      const queuedMessages = ([] as RunDagsterRunEventFragment[]).concat(
+        ...queuedLogs.map((log) => log.messages),
+      );
```

Comment:
Minor nit but I think this line is just doing `queuedLogs.flatMap(log => log.messages)` and we have that now that we're targeting ES2019?

Reply:
Yeah it is, good call!